### ISSUE TITLE
difference in game power by 1 % at least

### ISSUE THREAD

**Issue by filosgr** [cmmcleod/coriolis#116]
the bellow shows that i'am overpowering the ship , while in game i can run it

{
  "$schema": "http://cdn.coriolis.io/schemas/ship-loadout/2.json#",
  "name": "filos clipper1",
  "ship": "Imperial Clipper",
  "references": [
    {
      "name": "Coriolis.io",
      "url": "http://coriolis.io/outfit/imperial_clipper/36A6A5A5D6A5D4C2b2727270404040m4k3e5d5d5858246b.AwRhrSs1eYg%3D.CwBhrSs4CYEYGiA%3D?bn=filos%20clipper1",
      "code": "36A6A5A5D6A5D4C2b2727270404040m4k3e5d5d5858246b.AwRhrSs1eYg=.CwBhrSs4CYEYGiA=",
      "shipId": "imperial_clipper"
    }
  ],
  "components": {
    "standard": {
      "bulkheads": "Mirrored Surface Composite",
      "cargoHatch": {
        "enabled": false,
        "priority": 5
      },
      "powerPlant": {
        "class": 6,
        "rating": "A",
        "enabled": true,
        "priority": 1
      },
      "thrusters": {
        "class": 6,
        "rating": "A",
        "enabled": true,
        "priority": 1
      },
      "frameShiftDrive": {
        "class": 5,
        "rating": "A",
        "enabled": false,
        "priority": 5
      },
      "lifeSupport": {
        "class": 5,
        "rating": "D",
        "enabled": true,
        "priority": 1
      },
      "powerDistributor": {
        "class": 6,
        "rating": "A",
        "enabled": true,
        "priority": 1
      },
      "sensors": {
        "class": 5,
        "rating": "D",
        "enabled": true,
        "priority": 1
      },
      "fuelTank": {
        "class": 4,
        "rating": "C",
        "enabled": true,
        "priority": 1
      }
    },
    "hardpoints": [
      {
        "class": 3,
        "rating": "B",
        "enabled": true,
        "priority": 1,
        "group": "Plasma Accelerator",
        "mount": "Fixed"
      },
      {
        "class": 2,
        "rating": "F",
        "enabled": true,
        "priority": 1,
        "group": "Multi-cannon",
        "mount": "Gimballed"
      },
      {
        "class": 2,
        "rating": "F",
        "enabled": true,
        "priority": 1,
        "group": "Multi-cannon",
        "mount": "Gimballed"
      },
      {
        "class": 2,
        "rating": "F",
        "enabled": true,
        "priority": 1,
        "group": "Multi-cannon",
        "mount": "Gimballed"
      }
    ],
    "utility": [
      {
        "class": 0,
        "rating": "A",
        "enabled": true,
        "priority": 1,
        "group": "Shield Booster"
      },
      {
        "class": 0,
        "rating": "A",
        "enabled": true,
        "priority": 1,
        "group": "Shield Booster"
      },
      {
        "class": 0,
        "rating": "A",
        "enabled": true,
        "priority": 1,
        "group": "Shield Booster"
      },
      {
        "class": 0,
        "rating": "D",
        "enabled": true,
        "priority": 1,
        "group": "Kill Warrant Scanner"
      }
    ],
    "internal": [
      {
        "class": 7,
        "rating": "A",
        "enabled": true,
        "priority": 1,
        "group": "Shield Generator"
      },
      {
        "class": 6,
        "rating": "C",
        "enabled": false,
        "priority": 5,
        "group": "Fuel Scoop"
      },
      {
        "class": 4,
        "rating": "A",
        "enabled": true,
        "priority": 3,
        "group": "Shield Cell Bank"
      },
      {
        "class": 4,
        "rating": "A",
        "enabled": false,
        "priority": 2,
        "group": "Shield Cell Bank"
      },
      {
        "class": 3,
        "rating": "A",
        "enabled": true,
        "priority": 3,
        "group": "Shield Cell Bank"
      },
      {
        "class": 3,
        "rating": "A",
        "enabled": false,
        "priority": 2,
        "group": "Shield Cell Bank"
      },
      {
        "class": 1,
        "rating": "E",
        "enabled": false,
        "priority": 5,
        "group": "Docking Computer",
        "name": "Standard Docking Computer"
      },
      {
        "class": 2,
        "rating": "A",
        "enabled": false,
        "priority": 1,
        "group": "Frame Shift Drive Interdictor"
      }
    ]
  },
  "stats": {
    "class": 3,
    "hullCost": 21077780,
    "speed": 300,
    "boost": 380,
    "boostEnergy": 21,
    "agility": 2,
    "baseShieldStrength": 180,
    "baseArmour": 486,
    "hullMass": 400,
    "masslock": 12,
    "pipSpeed": 0.1,
    "shipCostMultiplier": 0.9,
    "componentCostMultiplier": 0.9,
    "fuelCapacity": 16,
    "cargoCapacity": 0,
    "ladenMass": 756.3,
    "armourAdded": 0,
    "armourMultiplier": 1.95,
    "shieldMultiplier": 1.6,
    "totalCost": 153670923,
    "unladenMass": 740.3,
    "totalDps": 16,
    "armour": 945,
    "powerAvailable": 25.2,
    "powerRetracted": 21.04,
    "powerDeployed": 25.33,
    "unladenRange": 16.53,
    "fullTankRange": 16.29,
    "ladenRange": 16.29,
    "unladenTotalRange": 57.8,
    "ladenTotalRange": 57.8,
    "maxJumpCount": 4,
    "shieldStrength": 489.6,
    "topSpeed": 344.85,
    "topBoost": 436.81
  }
}

**Comment by cmmcleod** [comment]
can you provide some screenshots of the modules/power management in game?

**Comment by filosgr** [comment]
There u go 

http://imgur.com/a/iT6sR

**Comment by cmmcleod** [comment]
Ok, so there are a couple of differences between the screenshots provided and the build you gave.

Using your screenshots I created [this build](http://coriolis.io/outfit/imperial_clipper/36A6A5A5E6E5D4C2b2727270404040m4k3e5d5d5858-6b.Iw19gDM3EA==.CwBhrSs4GYCYHGEA?bn=filos%20clipper%20%5Bscreen%20shot%5D)

Life Support: 6E (not 6A, but that uses less power)
Sensors: 5E (not 5D, but that uses less power)
No Docking Computer (again less power)

Power management was different between the build and the screen shots (disabled vs not disabled). This has been reflected in the build above. 

Power output of this build is 107.8% which matches in-game (In game power percentages are rounded down, I have tested this throughly).
